docs/SME.md

```diff
@@ -324,6 +324,56 @@ Cons:
   SME.method2();
   // Streaming mode stopped
   ```
+- Mixing non-streaming and streaming APIs in the same method would be valid but would probably result in bad performance. It will not be obvious to a user why this is happening.
```

Comment:
I am still little confused on the intention of "streaming_compatible" concept and wondering if we should bring that in .NET or just avoid it.